spec/google_visualr/app/helpers/view_helper_spec.rb

```diff
@@ -1,25 +1,23 @@
-require 'spec_helper'
-
 describe "ApplicationController" do
   describe "#render_chart" do
     it "has method" do
-      ApplicationController.instance_methods.should include :render_chart
+      expect(ApplicationController.instance_methods).to include :render_chart
     end
 
     it "includes method in corresponding helper" do
-      ApplicationController.helpers.methods.should  include :render_chart
+      expect(ApplicationController.helpers.methods).to  include :render_chart
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.